src/TLuaInterpreter.h

```diff
@@ -233,7 +233,7 @@ Q_OBJECT
     static int Send( lua_State * L );
     static int sendRaw( lua_State * L );
     static int Echo( lua_State * L );
-    static int select( lua_State * L );
+    static int selectString( lua_State * L ); // Was select but I think it clashes with the Lua command with that name
```

Comment:
The fact that it was this:

```
-    lua_register( pGlobalLua, "selectString", TLuaInterpreter::select );
+    lua_register( pGlobalLua, "selectString", TLuaInterpreter::selectString );
```

Means it did not clash, as "selectString" is what was registered with Lua. So the comment is unnecessary, but I welcome the change for consistency anyway.